hera_mc/tests/test_dataview.py

```diff
@@ -2,55 +2,43 @@
 # Copyright 2016 the HERA Collaboration
 # Licensed under the 2-clause BSD license.
 
-"""Testing for `hera_mc.connections`.
-
-"""
+"""Testing for `hera_mc.connections`."""
 
 from __future__ import absolute_import, division, print_function
 
```

Comment:
This module actually goes away in my following PR, so file gets removed.

Reply:
ok, I'll take it out in the rebase.